content/advent-2016/gobridge-beginners.md

```diff
@@ -0,0 +1,93 @@
+
+I did, and I’m here to talk about this. What I hope to achieve with this post is to introduce GoBridge, engage new instructors with existing communities and inspire new communities to start. 
+
+Let’s start with the brownie points: by teaching Go to complete beginners, not only you are getting involved in the Go community, you are also helping it grow. That’s a great thing to do as a supporter of an open-source community-driven programming language.
+
+# Meanwhile in Berlin
+In September-November 2016 we held a GoBridge workshop, where aspiring gophers met enthusiastic instructors every other Wednesday during 6 meetings in the SoundCloud offices in Berlin. The end result was a web server that pings a weather API and displays the results locally. The crowd was a great mix of uni students, people who already work at a different field, and even several developers. The mix of genders, religions and nationalities was as varied as it can get in Berlin, both at the students and at the instructors groups. 
+
+
+![](/postimages/advent-2016/gobridge-beginners-221216/golang_berlin.jpg)
+
+A few words on GoBridge: ths is an organization advocating for diversity in the Go community by educating underrepresented communities technical skills. GoBridge are the newest member of the BridgeFoundry umbrella, which is a great place to learn Ruby or mobile development. Alternatively, you can use go-mobile, but that's a subject for another post. If you have attended GolangUK '16 you probably remember Carlisia's [talk about GoBridge](https://www.youtube.com/watch?v=ImF_jDpjFNw).
+
+Back to our workshop. As we took it slow, the group of already-developers didn’t proceed to our second meeting. I hope they are reading this post today after being convinced Go is great. The first workshop already introduced several challenges, as developers want to plow through the material, but, we are here for beginners. So here is challenge #1 for you: control the pace to make the beginners feel comfortable with the learning pace.
+
+
+![](/postimages/advent-2016/gobridge-beginners-221216/flying_gopher.jpg)
+
+
+As the meetings went on, we covered simpler concepts like variables, and more complex one like json marshaling. We spotted an interesting behaviour: when the instructor leading the workshop asked “all clear?” there are no questions from the crowd. But, when s/he’d then say “let’s do this exercise” - not much happened. This was of course unclear, but no one raised their hand. Having taught beginner groups in the past, this was not new. Challenge #2: Get your crowd to actually understand. This can be broken down to 2 challenges:
```

Comment:
"This was of course unclear, but no one raised their hand." 

I think with this you are trying to say that students didn't clearly understand a concept, but they still weren't asking questions, but it is a little unclear. Perhaps it could be rephrased? I might try something like:

"This implied that everything wasn't clear to the students, but students still weren't raising their hands."